### PR TITLE
Project done

### PR DESCRIPTION
**Pull request by webmarkyn**
Finished Restaurant page project.
Used npm, webpack, import/export features.
Every DOM element is separated into its own module that's a function with returns a DOM element with styles and HTML and then get's added to page's DOM dynamically.
For linting used ESLINT

### CODE REVIEW

**Comment by webmarkyn** [comment]
Please run "npm start" to build the project. Then you can open index.html file

File: src/article.component.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+export default function article() {
+  const articleEl = document.createElement('articleEl');
+  articleEl.style.cssText = `
```
Comment (bellom) [inline src/article.component.js]: Let try as much as possible to keep our styling in CSS files, i believe we are all trying hard to practice good way of writing code. 👍 
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/

File: src/article.component.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    reiciendis sint et veritatis quo perferendis ullam voluptate quisquam
+    incidunt deserunt! Dolorem, quidem? Tenetur mollitia, vel soluta
+    tempore in cupiditate odit quis voluptatum debitis, ullam officia ad
+    error provident quia deserunt atque delectus quaerat voluptas fuga
+    incidunt ex aperiam aut illo sit! Ea a, ipsa ipsum ducimus dolore at
+    atque impedit neque fugiat ab illo quae repellat dicta itaque
+    consequuntur, doloremque aspernatur omnis maxime cumque magni. Debitis
+    corporis deleniti iusto aspernatur aperiam nesciunt, mollitia alias
+    nisi provident obcaecati?`;
+  p.style.cssText = `
```
Comment (bellom) [inline src/article.component.js]: Let try as much as possible to keep our styling in CSS files, i believe we are all trying hard to practice good way of writing code. 👍 
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/

File: src/header.component.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+export default function header() {
+  const headerEl = document.createElement('headerEl');
+  headerEl.style.cssText = `
+        width: 100%;
```
Comment (bellom) [inline src/header.component.js]: Let try as much as possible to keep our styling in CSS files, i believe we are all trying hard to practice good way of writing code. 👍 
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/

File: src/tabs.component.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+export default function tabs() {
+  const tabsEl = document.createElement('div');
+  tabsEl.style.cssText = `
```
Comment (bellom) [inline src/tabs.component.js]: Let try as much as possible to keep our styling in CSS files, i believe we are all trying hard to practice good way of writing code. 👍 
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @webmarkyn  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1:, just few issues left to work on............ :slightly_smiling_face: 

Below are part of the checklist for this project, kindly work on these few things listed below;
1. 
It features a tabbed menu. When a tab is clicked the content is replaced ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page))
image
As you can see the above instructions, once you clicked on one tab, it will wipe out the content not just part of the content.
**Screenshot from your web app below;**
image

**Hint:** Below screenshot is what we call tab menu in this project context.
image

Example of tabbed browsing is also shown here 👇 
image
https://www.hongkiat.com/blog/50-nice-clean-css-tab-based-navigation-scripts/


2. Kindly work on my comments below


**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by bellom — APPROVED** [review]
## Hi Team :clap: 

**Great work making all the necessary corrections from the previous review.**

**I trust you can improve the app, by using the tabbed browsing style used as an example in Odin page.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Goodluck in the next project :slightly_smiling_face:_ 

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**